Makefile

```diff
@@ -163,6 +163,8 @@ DEFENTROPYSOURCE := /dev/urandom
 DEFDISABLEBLOCK := false
 DEFVIRTIOFS := false
 DEFVIRTIOFSDAEMON :=
+# Default DAX mapping cache size in GiB
```

Comment:
Maybe `MiB` would be more appropriate if we don't want a too large cache region by default.

Reply:
Fixed in the next revision.